### PR TITLE
88 paperclip vulnerability

### PR DESCRIPTION
**Pull request by btpnlsl**
Updates gemfiles and FAQ

### CODE REVIEW

File: app/views/home/faq.html.erb [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-<strong><br/><p>What do I need in order to listen?</strong><br/>
-A computer or mobile device hooked up to the internet that you can plug speakers or headphones into.
-That's it!
+For ten years, Hollow Earth Radio was a live streaming internet station only.
+In 2013, long-time online broadcaster Hollow Earth Radio was approved for an LPFM license for the frequency 104.9 FM, with the call-letters KHUH!?
+Starting in 2017, after lengthy planning and fundraising campaign, our FM signal now emanates from the Central Area of Seattle.
 
-<strong><br/><p>How do I listen to Hollow Earth Radio?</strong><br/>
+<strong><br/><p/>What do I need in order to listen?</strong><br/>
+If you are in the Central Area or beyond, try tuning in with your radio!
```
Comment (btpnlsl) [inline app/views/home/faq.html.erb]: perhaps: try tuning into 104.9 FM

File: app/views/home/faq.html.erb [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+
+Anywhere else, we should be reachable via  computer or mobile device hooked up to the internet that you can plug speakers or headphones into.
+
+<strong><br/><p/>How do I listen to Hollow Earth Radio?</strong><br/>
```
Comment (btpnlsl) [inline app/views/home/faq.html.erb]: Do we need this header? or can we just have 1 'how do I listen' section

File: app/views/home/faq.html.erb [hunk 3]
```diff
@@ -46,65 +48,61 @@ You should stop by sometime, if the light is on someone is there so just knock a
...
 
-<strong><br/><p>Are you broadcasting live? </strong><br/>
+<strong><br/><p/>Are you broadcasting live? </strong><br/>
 
 We have DJs with weekly or bi-weekly <%= link_to "shows", controller: "programs"%> who spin music live and handle your requests.
 Check out our <%= link_to "calendar", "schedule"%> for schedules.
 When there is no live DJ, we have pre-recorded programs, local performances, archived radio shows, and
 unique mixes of music that we’ve crafted just for you.
 
-<strong><br/><p>Who started the radio & how long has it been around? </strong><br/>
```
Comment (btpnlsl) [inline app/views/home/faq.html.erb]: &amp;

Comment (btpnlsl) [reply app/views/home/faq.html.erb]: need to replace all & with `&amp;`

**Comment by boontdustie** [comment]
You and your code reviews!

**Pull request merged**